example-app/SantokuApp/src/features/demo-map/pages/DemoMapPage.tsx

```diff
@@ -0,0 +1,267 @@
+  {formKey: 'latitude', description: '緯度'},
+  {formKey: 'longitude', description: '経度'},
+];
+
+export const DemoMapPage: React.FC = () => {
+  const [defaultMarker, setDefaultMarker] = useState<MarkerProps>(initialMarker);
+  const [formSubmittedMarker, setFormSubmittedMarker] = useState<MarkerProps>();
+
+  const [region, setRegion] = useState<Region>(initialRegion);
+
+  const [mapType, setMapType] = useState<MapType>('standard');
+  const [showBuildings, setShowBuildings] = useState<boolean>(true);
+  const markerList = useMemo(
+    () => (formSubmittedMarker ? [defaultMarker, formSubmittedMarker] : [defaultMarker]),
+    [defaultMarker, formSubmittedMarker],
+  );
+
+  const onSubmit = useCallback(
+    (values: RegionFormValues) => {
+      const latitude = Number(values.latitude);
+      const longitude = Number(values.longitude);
+      const latitudeDelta = Number(values.latitudeDelta);
+      const longitudeDelta = Number(values.longitudeDelta);
+      setRegion({latitude, longitude, latitudeDelta, longitudeDelta});
+      setDefaultMarker({coordinate: {latitude, longitude}});
+    },
+    [setRegion],
+  );
+  const {form} = useRegionForm({
+    initialValues: {
+      latitude: initialRegion.latitude.toString(),
+      longitude: initialRegion.longitude.toString(),
+      latitudeDelta: initialRegion.latitudeDelta.toString(),
+      longitudeDelta: initialRegion.longitudeDelta.toString(),
+    },
+    onSubmit,
+  });
+  const onMarkerFormSubmit = useCallback((values: MarkerFormValues) => {
+    // 緯度と経度が入力されていた場合のみマーカーを追加し、そうでない場合はマーカーをクリアする
+    if (values.latitude && values.longitude) {
+      const marker = {
+        coordinate: {
+          latitude: Number(values.latitude),
+          longitude: Number(values.longitude),
+        },
+        title: values.title,
+        description: values.description,
+        draggable: values.draggable,
+      };
+      setFormSubmittedMarker(marker);
+    } else {
+      setFormSubmittedMarker(undefined);
+    }
+  }, []);
+  const {form: markerForm, setMarkerFormDraggable} = useMarkerForm({onSubmit: onMarkerFormSubmit});
+
+  const keyboardType = Platform.select({
+    ios: 'numbers-and-punctuation',
+    android: 'number-pad',
+  } as const);
+
+  const [scrollEnabled, setScrollEnabled] = useState<boolean>(true);
+  const [zoomEnabled, setZoomEnabled] = useState<boolean>(true);
+  const [rotateEnabled, setRotateEnabled] = useState<boolean>(true);
+  const [pitchEnabled, setPitchEnabled] = useState<boolean>(true);
+
+  const mapViewRef = useRef<MapViewRef>(null);
+
+  useEffect(() => {
+    mapViewRef.current?.animateToRegion(region);
+  }, [region]);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.flexContainer}>
+        <ScrollView>
+          <View style={styles.flexContainer}>
+            <SpecAndSourceCodeLink feature="map" />
+            <Text>
+              {`
+地図を表示します。オプションを変更し、表示や操作を確認できます。
+        `}
+            </Text>
```

Comment:
```suggestion
            <Text>地図を表示します。オプションを変更し、表示や操作を確認できます。</Text>
```

※ 改行が消えてレイアウトが変わります。

| 変更前 | 変更後 |
|:-------:|:-------:|
|<img src="https://github.com/ws-4020/mobile-app-crib-notes/assets/13145001/5d01fcad-2c0c-4a3d-9f8c-9a67a8d2414c"> | <img src="https://github.com/ws-4020/mobile-app-crib-notes/assets/13145001/0c5f56ab-aff8-4143-ba79-a38e62f355e4"> |